cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/BaseController.java

```diff
@@ -114,22 +115,28 @@ public static void main(String[] args) {
             } else if (ex instanceof ApplyException) {
                 log.error("An error occurred during the apply:", ex.getMessage());
             } else if (ex instanceof GetException) {
-                log.error("An error occurred during the get:", ex.getMessage());
-            } else if (ex instanceof IllegalStateException) {
-                // a little bit ugly, but it works
-                // the problem is in reactor.core.Exceptions
-                // it creates lambda class instances which are hard to test on...
-                if (ex.getMessage().toLowerCase().contains("retries exhausted")) {
-                    // now, if there is a cause and it's an InvalidTokenException, we can at least provide a hint
-                    // that it's likely a password issue
-                    if (ex.getCause() != null &&
-                            ex.getCause().toString().toLowerCase().contains("invalidtokenexception")) {
-                        log.error("Request to CF API failed: invalid token error (is the password correct?)");
-                    } else {
-                        log.exception(ex, "Request to CF API failed:");
-                    }
+                Throwable getExceptionCause = ex.getCause();
```

Comment:
The variable name doesn't make a lot of sense to me. Why not just call it "cause"?

Reply:
Will be renamed

Reply:
I see, but that doesn't get clear from the name. At first I was a bit confused because the call you make is to `getCause()` on some exception. I don't mind either way.